### ISSUE TITLE
[Feature Request] Remove "_." prefix when displaying or accessing variables

### ISSUE THREAD

**Issue by hsul4n** [getinsomnia/insomnia#3064]
Hello everybody,

I've been using insomnia for along time, but I have an issue with some workspaces:

1- When declaring an env variable I was able to access them using `{{name}}` but now I always got `{{_.name}}`.

2- I was able to use nested variable declaration for example:
```
Workspace
│
└─ Environments:  
│
└── Development (Sub-Env) // json: {"base_url": "https://example.com"}
│
└─ Folders
│
└── API (Folder) // json: {"base_url": "{{base_url}}/api"}
│
└─── V1 (Folder) // json:  {"base_url": "{{base_url}}/v1"}
│
└──── // etc..

// At the end we need to get full joined url for example:
// {{base_url}}: https://example.com/api/v1/etc..
```

In some workspaces I got the correct result but in other I got something like:
```
http:///%7B%7B%20_.base_url%20%7D%7D/v1/v1/v1/v1/v1/v1/v1/v1
```

I've export data and reinstall app and import data again but same problem.

Any suggestions!

**Comment by dcatt-haiku** [comment]
I have been running into this issue as well when defining new variables and have yet to make the **_.** prefix to go away.

**Comment by smacktoid** [comment]
Yeah I've been encountering this too. It seemed to come up when one of the latest versions was released.

Whenever I use ctrl + space and select a variable that I have defined in one of my environments it is rendered with this prefix. The workspace document has the prefix in it too.

It's problematic because Insomnia Designer doesn't know how to resolve the _. prefix that is being attached, so you can't import a workspace document from core to the designer.

I've manually removed the prefixes and Insomnia continues to work correctly.

**Comment by kedodrill** [comment]
I have this problem as well on Void Linux using the latest stable Insomnia Core (2020.5.2). Everything seems to work, it's just that it displays that way for some reason.

**Comment by dcatt-haiku** [comment]
Any plans on fixing this issue or are we stuck with it?

**Comment by develohpanda** [comment]
Hey folks, this was an intentional change that came within 2020.5, to give users the ability to [use special characters such as hyphens](https://github.com/Kong/insomnia/issues/2601) in environment variables. This should be fully backwards compatible however, as @scottmmmm noticed, it still works by removing the prefix.

Note that Designer has now been deprecated and merged into a single app, Insomnia, which should be able to handle the `_.` prefix correctly, everywhere.

Manually removing the prefix isn't a nice experience, so if the `_.` prefix is still causing breaking behavior on Insomnia v2021.1.1, could you please help me reproduce it by providing an example workspace to test with? 🙏🏽 

We've had some feedback about how it looks visually as well, so we may look to change it in the future, but for the time being a decision was made to prefix all (new) environment variables with the prefix, for consistency going forwards. 🤗

**Comment by smacktoid** [comment]
Thanks for commenting on this. It's good to understand why this was changed. 

Personally, I think the `_.` prefix looks a little messy and makes things a bit less readable, so I'd definitely +1 any improvements that can be made there. 

I like the idea of using hyphens in variables though.

**Comment by develohpanda** [comment]
No worries! I hear you, especially with readability. I'll see what we can do as it's evident the display needs to be revisited. 😃

**Comment by Kyle-sandeman-mrdfood** [comment]
Hi, just chipping in that almost a year later, there is still a big confusion for me (new user) on why autocomplete inserts `_.` when my env vars do not specify them. Even more confusing is why it still works without the prefix.. It leads me to believe the prefix should just be removed from insomnia? At minimum the docs should briefly cover why it exists
